daq/test_modules/base_module.py

```diff
@@ -30,6 +32,11 @@ def __init__(self, host, tmpdir, test_name, module_config):
         self.params = None
         self.start_time = None
 
+    def get_logger(self, log_name):
```

Comment:
This function can probably just stay in ipaddr_module for now. When there's a need for other in line modules, we can create a new InlineModule parent class to host this function. Storing it in base_module gives the illusion that all test modules use this extra logger to write out activate.log where in fact only the inline modules do.

Reply:
The goal was to get it out of ipaddr since I know it's not ipaddr-specific, and it does have more general utility. There are potentially reasons why a regular module might want to use this "to a host specific file" facility, not just for inline modules I'd wager. To avoid confusion with activate.log (which was confusing to me as well even just for ipaddr), I renamed it to module.log. (Yes, I know that functionally for ipaddr this is somewhat the same as activate.log -- but I think it's ultimately more confusing to have activate.log in two places.)